testdriver/test.yml

```diff
@@ -100,8 +100,7 @@ steps:
   - prompt: >-
       click on search bar and search for "rotate", click on it in the drop down,
       you will be taken to the rotate pdf page, if so then go back to the
-      homepage and clear the search bar present at the middle of the homepage and make sure all the tools are
```

Comment:
did it not work? :')

Reply:
ya so the issue was it clicked on the search bar in the taskbar a couple of times, so we felt it would be better to remove this and make sure that it always does the correct stuff .
So we are currently working on to spin up the VMs with auto-hide taskbar by default and will add this back once thats done.